frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segments_view.tsx

```diff
@@ -209,6 +218,45 @@ const mapStateToProps = (state: OxalisState): StateProps => {
   };
 };
 
+const getOrInitializeSelectedSegmentsOrGroup = (
```

Comment:
I'd rename this function to `getCleanedSelectedSegmentsOrGroup`, since I think the cleaning up part is more important than the initialization part.

Reply:
and also the initialization moved to dataset_reducer